examples/workflow/nextjs/README.md

```diff
@@ -21,7 +21,7 @@ At this point, if you are using a released `@upstash/qstash` version, you can de
 ```
 cd ../../..
 
-npm install
+bun install
```

Comment:
This was intentional, wanted to switch to bun since we have it's lock file in Qstash-js. Also I remember that it caused issues when you tried with npm, it was fixed when we changed to bun

I can change back if you think it's not necessary